src/styles/product.scss

```diff
@@ -0,0 +1,144 @@
+.product {
+  background-color: $page-backgroun-color;
+  overflow: scroll;
+
+  &__top {
+    padding: 15px 0 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__title {
+    letter-spacing: 0.9px;
+    text-transform: uppercase;
+    font-weight: 600;
+    font-size: 18px;
+    margin-bottom: 10px;
+
+    @include tablet {
+      display: none;
+    }
+  }
+
+  &__title-tablet {
+    display: none;
+
+    @include tablet {
+      display: block;
+      letter-spacing: 0.9px;
+      text-transform: uppercase;
+      font-weight: 600;
+      font-size: 18px;
+      margin-bottom: 25px;
+    }
+  }
+
+  &__price-ubtan {
+    display: none;
+
+    @include tablet {
+      display: block;
+      letter-spacing: 0.9px;
+      text-transform: uppercase;
+      font-weight: 500;
+      font-size: 18px;
+      margin-bottom: 10px;
+      color: $text-minor-color;
+    }
+  }
+
+  &__load {
+    display: none;
+
+    @include tablet {
+      display: block;
+      letter-spacing: 0.9px;
+      text-transform: uppercase;
+      font-weight: 500;
+      font-size: 18px;
+      margin-bottom: 30px;
+    }
+  }
+
+  &__price {
+    font-size: 14px;
+    line-height: 130%;
+    letter-spacing: 0.7px;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+
+    @include tablet {
+      display: none;
+    }
+  }
+
+  &__bottom {
+    @include tablet {
+      padding: 0 124px;
+    }
+  }
+
+  &__image {
```

Comment:
The use of a specific image in the CSS can limit the reusability of this class. Consider passing the image as a parameter from the HTML or using a more generic class name.